driver-core/src/test/functional/com/mongodb/client/CommandMonitoringTestHelper.java

```diff
@@ -218,7 +218,10 @@ private static CommandStartedEvent massageActualCommandStartedEvent(final Comman
         if (command.containsKey("recoveryToken")) {
             command.remove("recoveryToken");
         }
-
+        // Tests expect maxTimeMS to be int32, but Java API requires maxTime to be a long.  This massage seems preferable to casting
```

Comment:
I don't remember why I added this... will investigate...

Reply:
Probably because the are just shorter form and don't use the `$numberLong` syntax because its easy and dynamic languages!